app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.tsx

```diff
@@ -159,35 +159,34 @@ const TableVisualizationRow: React.FC<{
   const { branch_data, is_control } = results;
 
   const metricData = branch_data[metricKey];
-  // const percent = branch_data[METRIC.USER_COUNT]["percent"];
-  // const userCountMetric =
-  //   branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE]["point"];
+  const percent = branch_data[METRIC.USER_COUNT]["percent"];
+  const userCountMetric =
+    branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE]["point"];
 
   const branchType = is_control ? VARIANT_TYPE.CONTROL : VARIANT_TYPE.VARIANT;
   branchComparison =
     branchComparison || dataTypeMapping[tableLabel][branchType];
-  // const { lower, upper, point, count } = metricData[branchComparison];
-  const { lower, upper } = metricData[branchComparison];
+  const { lower, upper, point, count } = metricData[branchComparison];
   const significance = metricData["significance"];
 
   let field;
   switch (displayType) {
-    // case DISPLAY_TYPE.POPULATION:
-    //   field = populationField(point, percent);
-    //   break;
+    case DISPLAY_TYPE.POPULATION:
+      field = populationField(point, percent);
+      break;
     case DISPLAY_TYPE.COUNT:
       field = countField(lower, upper, significance, metricName, tableLabel);
       break;
     case DISPLAY_TYPE.PERCENT:
     case DISPLAY_TYPE.CONVERSION_RATE:
       field = percentField(lower, upper, significance, metricName, tableLabel);
       break;
-    // case DISPLAY_TYPE.CONVERSION_COUNT:
-    //   field = conversionCountField(count, userCountMetric);
-    //   break;
-    // case DISPLAY_TYPE.CONVERSION_CHANGE:
-    //   field = conversionChangeField(lower, upper, significance);
-    //   break;
+    case DISPLAY_TYPE.CONVERSION_COUNT:
```

Comment:
Let's definitely add these two cases to Storybook, I'll add the population field one since that's used in my PR.